var/spack/repos/builtin/packages/spack/package.py

```diff
@@ -31,6 +31,7 @@ class Spack(Package):
 
     # Python (with spack python -i ipython support)
     depends_on('python', type='run')
+    depends_on('python@2.7:', type='run', when='@0.18.0:')
```

Comment:
This could be made much more specific. The next release will likely support 2.7 and 3.5-3.10. Right now we support 2.6-2.7 and 3.5-3.9.

Reply:
right now we support :3.9 in then sense: we do not test 3.10? Or is Spack broken on 3.10? I didn't add an upperbound assuming the former.

Reply:
We don't test 3.10 yet, although it would be a good idea to test. At least a few things usually break in each new version.

Reply:
Right, I think we can assume that it works until proven differently, then I'm happy to update the recipe